Entity/JobLog.php

```diff
@@ -107,7 +90,7 @@ public function __construct(
         // none of these values are mutable so are suitable to generate uuid
         // the 'added' makes it unique
         if (!$uuid) {
-            $uuid = $this->generateUuid($command, $topic, $added);
+            $uuid = $this->generateUuid($command, $topic, $this->added->format('U'));
```

Comment:
Can we change this to an actual UUID? just noticed its somekind of hash?

Reply:
That was my original idea, however we only generate UUID if it's not already defined.
We generate it here https://github.com/usemarkup/JobQueueBundle/blob/develop/EventSubscriber/LogConsoleCommandEventSubscriber.php#L66

I have no idea what's the format of it but I guess it's different from doctrine's UUID.

Reply:
It seems really complicated here, can't we use `Uuid::uuid4()->toString()` to generate it? 

>  UUID if it's not already defined.

Bit confused here, can we not make it a required? that always generate it outside 24/7 to reduce the need for such complex logic.

Reply:
Have you seen the link I pasted?

Reply:
> we only generate UUID if it's not already defined

Can we remove this part, and force it to always be defined?

Reply:
Generally having an optional argument that isnt optional is confusing, thats the goal of the comment here. Remove the internal logic (that is nasty) and always ensure the UUID comes from the outside... and do `Uuid::uuid4()->toString()` if required (outside)

Reply:
So when the CLI command is executed, how do you find respective log for it to mark it as complete / failed? 
Right now we do it by searching for log by command's uuid.

Reply:
Yeah we'd search by UUID that wouldn't change.

Reply:
I will PR to this branch a solution, might be easier Im perhaps not explaining very well.

Reply:
Ive completed the work here https://github.com/usemarkup/JobQueueBundle/pull/83